basic-algorithm-scripting/find-the-longest-word-in-a-string.js

```diff
@@ -0,0 +1,13 @@
+function findLongestWordLength(str) {
+  let longestWord = str.split(" ");
```

Comment:
change this variable name... when you apply split method, what happens?
A string is divided considering some criteria.
So it's not the longestWord it's an array of words so words would be a better name for this variable.

Reply:
Mude o nome da variavel
quando vc usa o metodo split, o retorno eh um vetor 
entao nesse caso o nome longestWord nao se aplica pq o que tem dentro dela nao eh isso e sim um vetor de palavras
Use nesse caso o nome words